mongodb/configure.sls

```diff
@@ -23,9 +23,9 @@ place_mongodb_config_file:
 
 {% if 'mongodb_primary' in grains['roles'] %}
 
-{% set replset_config = {'_id': salt['pillar.get']('mongodb:replset_name', 'rs0'), 'members': []} %}
+{% set replset_config = {'_id': salt.pillar.get('mongodb:replset_name', 'rs0'), 'members': []} %}
 {% if salt.pillar.get("mongodb:VAGRANT", false) %}
-{% do replset_config['members'].append({'_id': 0, 'host': '192.168.33.10:' + mongodb.port}) %}
+{% do replset_config['members'].append({'_id': 0, 'host': salt.pillar.get('mongodb:vagrant_host') + ':' + mongodb.port}) %}
```

Comment:
Fixed in new [commit](https://github.com/mitodl/mongodb-formula/pull/6/commits/ec1ab0a8a6b0ec9b896f919d6035398e6d0b5c80)